src/main/java/org/fundacionjala/coding/fizzbuzz/Kata.java

```diff
@@ -0,0 +1,53 @@
+package org.fundacionjala.coding.fizzbuzz;
+
+import java.util.StringJoiner;
+import java.util.stream.IntStream;
+
+/**
+ * Write a program that prints the numbers from 1 to 100. But for multiples of three print Fizz instead
+ * of the number and for the multiples of five print Buzz. For numbers which are multiples of both three
+ * and five print FizzBuzz.
+ */
+public final class Kata {
+    /**
+     * Empty constructor.
+     */
+    private Kata() {
+    }
+
+    /**
+     * @param number quantity of print numbers
+     * @return The whole String
+     */
+    public static String fizzBuzz(final int number) {
+
+        int[] numbers = IntStream.rangeClosed(1, number).toArray();
+        StringJoiner result = new StringJoiner("\n");
+        for (int a : numbers) {
+            result.add(changeFB(a));
+        }
+        return result.toString();
+    }
+
+    /**
+     * @param number Number to evaluate 3 or 5.
+     * @return return the number converted.
+     */
+    public static String changeFB(final int number) {
+        final int fizz = 3;
+        final int buzz = 5;
+        StringBuilder result = new StringBuilder();
```

Comment:
Is really need a StringBuilder here ?
Keep in mind that you are not using a loop.